tests/test_playbooks/katello_smart_proxy.yml

```diff
@@ -34,6 +34,18 @@
       vars:
         smart_proxy_state: present
         expected_change: false
+    - include_tasks: tasks/smart_proxy.yml
+      vars:
+        smart_proxy_state: present
+        smart_proxy_lifecycle_environments:
+          - 'Dev'
```

Comment:
You'll have to create this env like in https://github.com/theforeman/foreman-ansible-modules/blob/6a4c479bec608c3d430bb94d9e86a0e09ac5173b/tests/test_playbooks/content_view_version.yml#L18-L33

Also probably cool to have a tests that replaces Dev with Test (or smth similar, so we also test the "remove" branch)

Reply:
I made some changed to the tests, is that what you meant? :-) It now created 2 LCE's and tries to bind them one at a time.

Reply:
Yes, that's exactly what I wanted! Thanks!

Reply:
Awesome, nice!